lib/site_template/_config.yml

```diff
@@ -17,5 +17,12 @@ url: "http://yourdomain.com" # the base hostname & protocol for your site
 twitter_username: jekyllrb
 github_username:  jekyll
 
+# The configuration file also allows you to set site-wide variables that can be
+# accessed as `site.my_variable`. You'll notice in the default site layout,
+# these produce branded links to the corresponding Twitter and Github account.
+# The template `_include/footer.html` and `_include/icon-github.html` show you
+# how to use these: just reference the site.github_username to determine what
```

Comment:
Not sure what you mean by this and the following line. Would you please explain?